### PR TITLE
Use Ruff code and body as Diagnostic code and body

### PR DESCRIPTION
**Pull request by charliermarsh**
This more closely mirrors the behavior of other linters (i.e., the code appears in parentheses):

Screen Shot 2022-10-30 at 6 50 41 PM

### CODE REVIEW

File: ruffd-core/src/server_ops.rs [hunk 1]
```diff
@@ -13,8 +13,6 @@ use std::sync::Arc;
 // TODO macro the unwrapping of state_handles
 
 fn message_into_diagnostic(msg: Message) -> lsp_types::Diagnostic {
-    // As ruff currently doesn't support the span of the error,
-    // only have it span a single character
```
Comment (charliermarsh) [inline ruffd-core/src/server_ops.rs:18]: I believe these comments are stale.

Comment (Seamooo) [reply ruffd-core/src/server_ops.rs:18]: Definitely stale, there's more stale comments around I'll look to clean up, most notably around TODOs

File: ruffd-core/src/server_ops.rs [hunk 2]
```diff
@@ -31,10 +29,9 @@ fn message_into_diagnostic(msg: Message) -> lsp_types::Diagnostic {
         };
         lsp_types::Range { start, end }
     };
-    let code = Some(lsp_types::NumberOrString::String(msg.kind.body()));
```
Comment (Seamooo) [inline ruffd-core/src/server_ops.rs:35]: Thanks for catching this! I'm pretty sure it's a regression issue from when the code interface changed

**Comment by Seamooo** [comment]
Happy to merge after the small formatting fix

**Pull request merged**